nnvm/python/nnvm/frontend/tensorflow.py

```diff
@@ -252,8 +252,12 @@ def _impl(inputs, attr, params):
                 in_h = input_shape[2]
                 in_w = input_shape[3]
 
-            pad_v = _get_pad_pair(in_h, kernel_h, stride_h)
-            pad_h = _get_pad_pair(in_w, kernel_w, stride_w)
+            dilation_h = attr['dilations'][0]
```

Comment:
@Rasterer Can you explain this change ?

Reply:
Please refer to [https://github.com/dmlc/tvm/blob/master/nnvm/src/top/nn/convolution.cc#L100](url) for the affect of dilation on output size. Dilated conv always can be simulated by a normal conv with dilated kernel. So I use dilated kernel size for pad calculation instead.

Reply:
@srkreddy1238 any more comment about this?